examples/polypharmacy_gripnet/main.py

```diff
@@ -1,73 +1,60 @@
-import argparse
-import os
+import warnings
 
+import pytorch_lightning as pl
 import torch
 from config import get_cfg_defaults
-from model import GripNet
-from trainer import Trainer
+from model import GripNetLinkPrediction
```

Comment:
We prefer to use `get_model`. See action_dann, digits_dann, bindingdb_deepdta, office_multisource_adapt on how we config examples when using lightning.